public/src/main/java/com/revenuecat/purchases/models/PurchaseInfo.kt

```diff
@@ -0,0 +1,8 @@
+package com.revenuecat.purchases.models
+
+import com.revenuecat.purchases.ProductType
+
+interface PurchaseInfo {
```

Comment:
This doesn't need to be exposed so it can be `internal` I think.

Reply:
@tonidero I tried to change this to `internal` but things got angry because we have multiple modules 😬  This is in common but then its concrete versions live in Amazon and Google so I don't think that works the way I want it to 😔

Reply:
Ugh, yeah... I really would like to remove as many modules as possible in the future... For now, have you tried moving this interface to the `common` module? The concrete implementations could live in the `feature/google` and `feature/amazon` modules. It can be public in those three modules. 

This is because currently only the `public` and `purchases` modules expose functions in the API. We import the other modules using gradle's [`implementation`](https://developer.android.com/studio/build/dependencies#dependency_configurations) to setup the dependencies most of the time, which means we won't expose the public code in those dependencies.